provider/server/status_test.go

```diff
@@ -37,16 +37,6 @@ var _ = Describe("Status", func() {
 				},
 				Quantity: mcr.GetQuantity(&machineClasses[0], hostResources),
 			},
-			&iriv1alpha1.MachineClassStatus{
```

Comment:
why?

Reply:
As discussed offline, retaining this configuration can be beneficial for testing and closely resembles the production setup. Therefore, we have decided to keep it.

Reply:
reverted